include/mbedtls/mbedtls_config.h

```diff
@@ -1637,7 +1637,8 @@
 *
 * Enable support for RFC 8446 TLS 1.3 early data.
 *
-* Requires: MBEDTLS_SSL_TLS1_3_KEY_EXCHANGE_MODE_PSK_ENABLED or
+* Requires: MBEDTLS_SSL_SESSION_TICKETS and either
+*           MBEDTLS_SSL_TLS1_3_KEY_EXCHANGE_MODE_PSK_ENABLED or
```

Comment:
Comment on line 1651 below. As MBEDTLS_SSL_TLS1_3_KEY_EXCHANGE_MODE_* are enabled by default as well as MBEDTLS_SSL_SESSION_TICKETS I would suggest to do the same with MBEDTLS_SSL_EARLY_DATA. Then disable it in the tests when the requisite for MBEDTLS_SSL_EARLY_DATA are not met. Thus line 1651, rather:
```C
#define MBEDTLS_SSL_EARLY_DATA
```

Reply:
Haha, I never think this solution. Agree.
I think we can change `tls13-only.h` , but not very sure about that.